src/nodeDebugConfigurationProvider.ts

```diff
@@ -228,7 +228,7 @@ function createLaunchConfigFromContext(
           dir += '/';
         }
       }
-      config['preLaunchTask'] = 'tsc: build - tsconfig.json';
+      (config as any)['preLaunchTask'] = 'tsc: build - tsconfig.json';
```

Comment:
You can probably add this as `preLaunchTask?: string;` in IMandatedConfiguration